Lebenslauf-Website/index.html

```diff
@@ -90,22 +90,41 @@ <h2 class="text-white bg-black text-3xl pl-5 pr-5 pt-1 pb-2 mt-20 mb-9">
         <h2 class="text-white bg-black text-3xl pl-5 pr-5 pt-1 pb-2 mt-20 mb-9">
           Socials
         </h2>
-        <ul>
-          <li>
-            <a href="#"><img src="images/linkedin.png" alt="" />LinkedIn</a>
-          </li>
-          <li>
-            <a href="#"><img src="images/xing.png" alt="" />Xing</a>
-          </li>
-          <li>
-            <a href="#"><img src="images/github.png" alt="" />Github</a>
-          </li>
-        </ul>
+        <div
+          class="
+            grid grid-cols-3
```

Comment:
why do you use grid and flex together?

Reply:
the positioning here is for the following div containers AND their content. you know another way? i'd appriciate.